src/main/java/com/hrznstudio/titanium/block/tile/TileActive.java

```diff
@@ -66,7 +68,7 @@ public TileActive(BlockTileBase base) {
     @Override
     public boolean onActivated(PlayerEntity playerIn, Hand hand, Direction facing, double hitX, double hitY, double hitZ) {
         if (multiTankHandler != null) {
-            return FluidUtil.interactWithFluidHandler(playerIn, hand, multiTankHandler.getCapabilityForSide(null));
+            return FluidUtil.interactWithFluidHandler(playerIn, hand, multiTankHandler.getCapabilityForSide(null).orElse(new MultiTankHandler.MultiTankCapabilityHandler(new ArrayList<>())));
```

Comment:
What is the point of creating a new Handler here?

Reply:
The capability can't be null in that method